kairon/shared/channels/mail/processor.py

```
@@ -1,14 +1,18 @@
 import asyncio
 import time
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)